test/tls-provider.c

```diff
@@ -3220,12 +3220,12 @@ int tls_provider_init(const OSSL_CORE_HANDLE *handle,
     OSSL_LIB_CTX *libctx = OSSL_LIB_CTX_new_from_dispatch(handle, in);
     OSSL_FUNC_core_obj_create_fn *c_obj_create= NULL;
     OSSL_FUNC_core_obj_add_sigid_fn *c_obj_add_sigid= NULL;
```

Comment:
pre-existing: missing space in front of the `=` on this line and the one above. Up to you if you want to fix this (now we're in the area, which is my fault, sorry!)